annif/vocab.py

```diff
@@ -55,6 +55,18 @@ def subjects(self):
                     "subject file {} not found".format(path))
         return self._subjects
 
+    @property
+    def skos(self):
+        """return the subject vocabulary from SKOS file"""
+        if self._skos_vocab is None:
+            path = os.path.join(self.datadir, 'subjects.ttl')
+            if os.path.exists(path):
+                logger.debug(f'loading graph from {path}')
+                self._skos_vocab = annif.corpus.SubjectFileSKOS(path, None)
```

Comment:
Here the language for the SKOS vocabulary is set to `None` as the language is not available from within the `AnnifVocabulary` and it's currently not needed. But should the language be made available by passing it from `AnnifProject` when instantiating `AnnifVocabulary`?

Reply:
I guess that would be better...I'm a bit nervous about None values